public/index.html

```diff
@@ -11,7 +11,7 @@
     <noscript>
       <strong>We're sorry but Cylc UI doesn't work properly without JavaScript enabled. Please enable it to continue.</strong>
     </noscript>
-    <div id="app"></div>
+    <div id="app" data-base-url="{{ python_base_url }}"></div>
```

Comment:
Where does `python_base_url` come from?

Reply:
From Cylc UI Server. The `index.html` is rendered by Tornado. In the offline mode this value will be as-is, but it is ignored in JS.

Reply:
Aaah

Reply:
Added a comment explaining the above :+1: